android/src/app/organicmaps/util/Graphics.java

```diff
@@ -141,5 +141,26 @@ public static Drawable tint(Drawable src, ColorStateList tintColors)
     return res;
   }
 
+  @NonNull
+  public static Bitmap drawableToBitmap(@NonNull Drawable drawable)
+  {
+    if (drawable instanceof BitmapDrawable)
+    {
+      final BitmapDrawable bitmapDrawable = (BitmapDrawable) drawable;
+      if (bitmapDrawable.getBitmap() != null)
+      {
+        return bitmapDrawable.getBitmap();
+      }
```

Comment:
We don't use {} for one-liners.

Reply:
https://github.com/organicmaps/organicmaps/pull/4069